src/components/request/request.test.js

```diff
@@ -3,6 +3,11 @@ import { mount } from 'enzyme';
 import Request from '.';
 import accounts from '../../../test/constants/accounts';
 
+jest.mock('../../i18n', () => ({
+  t: v => v,
+  language: 'en',
+}));
+
```

Comment:
This looks like something that might be worth to do globally in `config/setupJest.js`. We are already mocking `i18next.t` in `config/setupJest.js:26`, so you just can mock also `i18next.init` that is used in `i18n/i18n-setup.js` to return objects with the language you want.